Microsoft.WindowsAzure.Storage/tests/cloud_storage_account_test.cpp

```diff
@@ -920,10 +922,15 @@ SUITE(Core)
             policy.set_resource_type((azure::storage::account_shared_access_policy::resource_types)0x7);
 
             policy.set_permissions((uint8_t)i);
+            std::cout << "i == " + std::to_string(i) + " here1\n" << std::flush;
```

Comment:
>std::cout << "i == " + std::to_string(i) + " here1\n" << std::flush; [](start = 12, length = 68)

I believe you forgot to remove the debug information.

Reply:
sure